README.md

```diff
@@ -52,12 +52,11 @@ For more information about IRMA, see: https://privacybydesign.foundation/irma/
 
 The IRMA client apps can be downloaded from their respective app stores:
 
-    Apple App Store
-    Google Play Store
+- [F-driod catalogue](https://f-droid.org/en/packages/org.irmacard.cardemu/)
```

Comment:
Thanks for contributing! This line still has a small typo. The correct line should be
```
[F-droid catalogue](https://f-droid.org/en/packages/org.irmacard.cardemu/)
```

Would you mind updating the PR with that line?

Reply:
The O and I are reversed in your commit. F-dr**oi**d (correct) vs F-dr**io**d. I'd be happy to add a commit myself, but you'd have to approve that PR on your branch, then it would be folded into this one.